src/main/java/net/rcarz/jiraclient/tempo/Team.java

```diff
@@ -176,8 +180,37 @@ public static boolean delete(RestClient restClient, Integer id) throws JiraExcep
 
     }
 
+    public List<User> getTeamMembers() throws JiraException {
```

Comment:
better to rename the method - "get..." is better to use for default getters - nobody will expect here HTTP request inside
you can rename it to find... + cache results in the field + add default getter/setter + add java doc for implementation